code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -45,6 +45,11 @@
 					movement_target = null
 					stop_automated_movement = 0
 					break
+			for(var/obj/item/toy/cattoy/T in view(1,src))
+				if (!T.cooldown)
+					emote("me", 1, "bats \the [T] around with its paw!")
+					T.cooldown = 1
+					spawn(100) T.cooldown = 0

```

Comment:
@KorPhaeron There's no real consensus, I personally prefer  ```world.time``` besides the fact it might explode at midnight rollover (but that might just be ```world.timeofday```